data/config.dist.json

```diff
@@ -5,6 +5,6 @@
     "array", "string", "int", "float", "bool", "iterable", "callable", "void"
   ],
   "php-core-extensions" : [
-    "Core", "date", "pcre", "Reflection", "SPL", "standard"
+    "Core", "date", "pcre", "Phar", "Reflection", "SPL", "standard"
```

Comment:
@maglnet can we split this line? Each diff will lead to destroying previous change history